src/context-selection/menu-select/menu-select.module.css

```diff
@@ -1,6 +1,6 @@
 .menuSelect {
     width: 400px;
-    max-height: 100vh;
+    max-height: 70vh;
```

Comment:
This could be 80vh to match the max-height that is used in org-unit-selector-bar-item.module.css (or they could both be set to 70vh)